en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,795 @@
+    
+        Click the **+** icon below the `queryParams`statement, and then click **Variable**.
+        
+        Enter the following information:
+    
+        | **Field**         | **Value**                    |
+        |-------------------|------------------------------|
+        | **Type**          | **string**                   |     
+        | **Name**          | `inventoryItemId`            |
+        | **Expression**    | `queryParams.get("item")[0]` |
+        
+        Click **Save**.
+        
+    3. To allow the service to identify the quantity ordered, define another variable.
+    
+        Click the **+** icon below the `inventoryItemId` variable statement, and then click **Variable**.
+        
+        Enter the following information:
+        
+        | **Field**         | **Value**                                           |
+        |-------------------|-----------------------------------------------------|
+        | **Type**          | **int**                                             |     
+        | **Name**          | `quantity`                                          |
+        | **Expression**    | `check int:fromString(queryParams.get("count")[0])` |
+        
+        Click **Save**.
+     
+ 5. To connect to the Google sheet where inventory records are maintained, add an API call as follows: 
+            
+    Click the **+** icon below the `quantity` variable statement, and then click **API Calls**. Then click **Google Sheets**.
+    
+    To access the specific account you need to use for this scenario, click **Connect to Google Sheets**, and then click **Proceed**. Click on the Google Account of your choice and click **Allow** to allow Choreo to access it as described on the page. Then click **Save**.
+    
+    Now you can configure the Google sheets Google account you selected to receive API calls. 
+    
+    To do this, click the **+** icon below the Google Sheets connection you just added. Click **API Calls**, and then click **sheetsEndpoint** under **Choose existing connection**.
+    
+    In the **Operation** field, select **Get values from a column**. Then enter the following information in the rest of the fields that appear:
+    
+    | **Field**                  | **Value**                                                                                                       |
+    |----------------------------|-----------------------------------------------------------------------------------------------------------------|
+    | **SpreadsheetId**          | The ID of your Google sheet. This can be derived from the URL of your Google sheet. e.g., If the Google Sheet URL is `https://docs.google.com/spreadsheets/d/1wHOdJgJzN6V9-vBQVSLEYxpYDp5XnpIcNqgj-h9pX4Q/edit#gid=0`, the google sheet ID is `1wHOdJgJzN6V9-vBQVSLEYxpYDp5XnpIcNqgj-h9pX4Q`. |                                                                                  |  
```

Comment:
Instead of giving an example with a real doc, can we follow an approach like below.
https://developers.google.com/sheets/api/guides/concepts#spreadsheet_id